### DISCUSSION TITLE
ERROR! Unhandled Exception: System.AccessViolationException: Attempted to read or write protected memory.

### DISCUSSION THREAD

**Discussion by ab4728** [bonsai-rx/bonsai#1813] [Q&A]
[Bonsai workflow.zip](https://github.com/bonsai-rx/bonsai/files/15394405/Bonsai.workflow.zip)

![thumbnail_IMG_4459 (1)](https://github.com/bonsai-rx/bonsai/assets/111145910/352bfe09-06a2-4455-8653-33d1a909780b)

Hi everyone, my lab is using Bonsai for a project and we have encountered the error code I have attached as a screenshot. If anyone has any idea how to handle it I would be glad to hear your advice. I have included the workflow in case it helps. 

The program is throwing this error as soon as I press start on the workflow. Note that it does run fine if I disable all of the nodes stemming from FP3001. 

A few things I have tried in my own debugging process to no avail:

1. Updated the .NET framework
2.  Ran a check for corrupted files through the command prompt menu and then restarted my computer upon repairing them.
3. Ran an antivirus scan on the computer.
4. Updated all Bonsai packages. 
5. Ran the program as an administrator in case there is an issue with permissions.

**Comment by bruno-f-cruz** [comment]
Since it seems like an error scoped to that specific package, I would probably ask the maintainers of the package for help.

https://github.com/neurophotometrics/neurophotometrics/issues

Also, in the future try to use titles that other users can more easily search for, as asked in #864. Thanks

**Reply by ab4728** [reply]
Sure thing. I have gone and posted my issue on the page you linked. Thanks!
